doc/gpumd/input_parameters/ensemble_heat.rst

```diff
@@ -0,0 +1,32 @@
+.. _kw_ensemble_heat:
+
+:attr:`ensemble` (thermal transport)
+====================================
+
+The :attr:`ensemble` keyword is used to set up an integration method (an integrator).
+The integrators described on this page provide methods for studying thermal transport via classical :term:`MD` simulations.
+Background information pertaining to the methods referred to on this page can be found :ref:`here <green_kubo_method>`.
```

Comment:
The `heat` versions are used for NEMD methods not Green-Kubo.

Reply:
Those methods are all on the same page and the reference is shared.